File: agent/src/main/java/pt/um/lei/masb/agent/data/feed/AdafruitPublishJSON.java
package pt.um.lei.masb.agent.data.feed;

public class AdafruitPublishJSON {
    private String value;
    private String lat;
    private String created_at;
    private String lon;
    //private String ele;

    public AdafruitPublishJSON(){

    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }

    public String getLat() {
        return lat;
    }

    public void setLat(String lat) {
        this.lat = lat;
    }

    public String getLon() {
        return lon;
    }

    public void setLon(String lon) {
        this.lon = lon;
    }

    public String getCreated_at() {
        return created_at;
    }

    public void setCreated_at(String created_at) {
        this.created_at = created_at;
    }
}

File: agent/src/main/kotlin/pt/um/lei/masb/agent/data/feed/AdafruitPublishJSON.kt
package pt.um.lei.masb.agent.data.feed

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable


@Serializable
data class AdafruitPublishJSON(
    var value: String = "",
    var lat: String = "",
    var lon: String = "",
    var alt: String = "",
    @SerialName("created_at")
    var createdAt: String = ""
)


